src/stratisd_client_dbus/_implementation.py

```diff
@@ -22,16 +22,20 @@
 
 from ._data import SPECS
 
+TIME_OUT = 120  # In seconds
+
 ObjectManager = make_class(
    "ObjectManager",
-   ET.fromstring(SPECS['org.freedesktop.DBus.ObjectManager'])
+   ET.fromstring(SPECS['org.freedesktop.DBus.ObjectManager']), TIME_OUT
```

Comment:
Please put ```TIME_OUT``` on new line, indented three spaces for this and all the rest.

Reply:
Done, although I'm not sure why 3 spaces instead of 4 like the rest of the code in the project...

Reply:
There is a consistent rule. If the indentation is semantic, it gets four spaces, if not, three.